networks/bajun/project.yaml

```diff
@@ -0,0 +1,90 @@
+specVersion: 0.2.0
+name: nova
+version: 1.0.0
+description: Nova SubQuery project
+repository: https://github.com/nova-wallet/subquery-nova
+schema:
+  file: ./schema.graphql
+network:
+  genesisHash: '0x0e06260459b4f9034aba0a75108c08ed73ea51d2763562749b1d3600986c4ea5'
+  endpoint: wss://rpc-parachain.bajun.network
+
+dataSources:
+  - name: main
+    kind: substrate/Runtime
+    startBlock: 1
+    mapping:
+      file: ./dist/src/index.js
+      handlers:
+        - handler: handleHistoryElement
```

Comment:
I don't think we need all this handlers now. Let's keep it simple and drop all related to relay chain staking for now